client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -0,0 +1,172 @@
+import PressThis from '../press-this';
+
+class PublishingTools extends Component {
+	componentDidUpdate() {
+		const {
+			fields,
+			postByEmailAddressModuleActive,
+			regeneratingPostByEmail,
+			selectedSiteId
+		} = this.props;
+
+		if ( postByEmailAddressModuleActive && regeneratingPostByEmail === null && ! fields.post_by_email_address ) {
+			this.props.regeneratePostByEmail( selectedSiteId );
+		}
+	}
+
+	onRegenerateButtonClick = () => {
+		this.props.regeneratePostByEmail( this.props.selectedSiteId );
+	}
+
+	isFormPending() {
+		const {
+			isRequestingSettings,
+			isSavingSettings,
+		} = this.props;
+
+		return isRequestingSettings || isSavingSettings;
+	}
+
+	renderPostByEmailSettings() {
+		const { fields, translate, postByEmailAddressModuleActive, regeneratingPostByEmail } = this.props;
+		const isFormPending = this.isFormPending();
+		const email = fields.post_by_email_address && fields.post_by_email_address !== 'regenerate' ? fields.post_by_email_address : '';
+		const labelClassName = regeneratingPostByEmail || ! postByEmailAddressModuleActive ? 'is-disabled' : null;
+
+		return (
+			<div className="publishing-tools__module-settings is-indented">
+				<FormLabel className={ labelClassName }>
+					{ translate( 'Email Address' ) }
+				</FormLabel>
+				<ClipboardButtonInput
+					className="publishing-tools__email-address"
+					disabled={ regeneratingPostByEmail || ! postByEmailAddressModuleActive }
+					value={ email }
+				/>
+				<Button
+					compact
+					onClick={ this.onRegenerateButtonClick }
+					disabled={ isFormPending || regeneratingPostByEmail || ! postByEmailAddressModuleActive }
+				>
+					{ regeneratingPostByEmail
+						? translate( 'Regenerating…' )
+						: translate( 'Regenerate address' )
+					}
+				</Button>
+			</div>
+		);
+	}
+
+	renderPostByEmailModule() {
+		const {
+			selectedSiteId,
+			translate
+		} = this.props;
+		const formPending = this.isFormPending();
+
+		return (
+			<FormFieldset>
+				<div className="publishing-tools__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href={ 'https://jetpack.com/support/post-by-email/' } target="_blank">
+							{ translate( 'Learn more about Post by Email' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="post-by-email"
+					label={ translate( 'Publish posts by sending an email.' ) }
+					disabled={ formPending }
+					/>
+
+				{ this.renderPostByEmailSettings() }
+			</FormFieldset>
+		);
+	}
+
+	renderPressThis() {
+		const { translate } = this.props;
+		if ( ! config.isEnabled( 'press-this' ) ) {
+			return null;
+		}
+
+		return (
+			<div>
+				<FormLegend>{ translate( 'Press This' ) }</FormLegend>
+				<PressThis />
+			</div>
+		);
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Publishing Tools' ) } />
```

Comment:
Alternate string suggestion: 
* `translate( 'Additional tools that make creating and publishing your content easy:' )`&mdash; translations: **3**. *ES Score: 5.05*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-settings-publishing-tools-card)